common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -235,9 +233,7 @@ private AuthorizationResult performAuthorizationRequest(@NonNull final OAuth2Str
                 mAuthorizationStrategy
         );
 
-        final AuthorizationResult result = future.get();
-
-        return result;
+        return future.get();
```

Comment:
Maybe we should add a timeout here?

Reply:
We should, but let's do that separately....